front/components/assistant_builder/AssistantBuilder.tsx

```diff
@@ -852,7 +853,56 @@ export default function AssistantBuilder({
           </div>
 
           <div className="flex flex-col gap-6 text-sm text-element-700">
-            <div className="text-2xl font-bold text-element-900">Actions</div>
+            <div className="text-2xl font-bold text-element-900">
+              Data Sources & Actions
+            </div>
+            {configurableDataSources.length === 0 && (
+              <ContentMessage title="You don't have any active data source or connection">
+                <div className="flex flex-col gap-y-3">
+                  <div>Assistants can answer using existing knowledge.</div>
+                  <div>
+                    There are two types of knowledge sources:{" "}
+                    <strong>Data Sources</strong> (Files you can upload) and{" "}
+                    <strong>Connections</strong> (Automatically synchronized
+                    with platforms like Notion, Slack, ...).
+                  </div>
+                  {(() => {
+                    switch (owner.role) {
```

Comment:
nit: case default asserNever could be nice for future us

Reply:
(done)

Reply:
Slightly updating the text:

```
<div>Assistants can incorporate existing company data and knowledge to formulate answers.</div>
```